wbooks/src/redux/store.ts

```diff
@@ -0,0 +1,5 @@
+import { createStore } from 'redux';
+import Reactotron from '@config/reactotronConfig';
+import bookReducer from '@redux/book/reducer';
+
+export default createStore(bookReducer, Reactotron.createEnhancer());
```

Comment:
bookReducer is one of many. The store should receive an array of reducers even if you only have one

Reply:
I put a combine reducer